native/c/test/zowex.ds.test.cpp

```diff
@@ -206,7 +203,8 @@ void zowex_ds_tests()
                              int rc = execute_command_with_output(command, response);
                              ExpectWithContext(rc, response).ToBe(0);
                              vector<string> tokens = parse_rfc_response(response, ",");
```

Comment:
(nit) It is safer to first assert the length of the vector before using `operator[]` to avoid exceptions when accessing indices in tests 😋 

Any `std::out_of_range` exception (from out-of-bounds array access) would not be caught, which may affect/skip subsequent tests, and any error details that percolate could be vague. Not suggesting any changes for this though, just something to keep in mind for future tests around CSV parsing 👍